tests/lib/rules/type-annotation-spacing.js

```diff
@@ -20,6 +20,14 @@ const ruleTester = new RuleTester();
 ruleTester.run("type-annotation-spacing", rule, {
     valid: [
         {
+            code: `
+interface resolve {
+    resolver: (() => PromiseLike<T>) | PromiseLike<T>;
+}
+            `,
+            parser: "typescript-eslint-parser"
```

Comment:
If you pass `{ parser: "typescript-eslint-parser" }` to the `RuleTester` constructor, it will include that option in all tests, which will let you just have an array of code blocks to test.

Reply:
Yeah, I saw this in one of your tests and though of doing the same for new rules.
We could probably create a new bug to refactor the tests, right now we have different styles and is driving me crazy... 

are you OK with leaving this as is for now? I will work on the refactor during the weekend

Reply:
> are you OK with leaving this as is for now?

Yes 👍